solc/CommandLineInterface.cpp

```diff
@@ -487,20 +491,26 @@ bool CommandLineInterface::parseLibraryOption(string const& _input)
 void CommandLineInterface::createFile(string const& _fileName, string const& _data)
 {
 	namespace fs = boost::filesystem;
+
 	// create directory if not existent
-	fs::path p(m_args.at(g_argOutputDir).as<string>());
```

Comment:
Please do not touch this function in this PR.

Reply:
@chriseth My apologies, I didn't mean to have those make it into the PR.  I will remove any touch to this function.